torch/_inductor/dependencies.py

```diff
@@ -734,6 +734,15 @@ def reduction(
         num_values = reduction_num_outputs(reduction_type)
         return (None,) * num_values if num_values > 1 else None
 
+    def masked(self, mask, body, other) -> None:
+        if callable(body):
+            # The body is typically a lambda which can make
+            # additional calls like ops.indirect_indexing.
+            try:
+                body()
+            except TypeError:
+                pass
```

Comment:
When are we getting a type error here?  Should this be an error?

Reply:
Yup, this should be an error.